src/org/typo3/chefci/helpers/JenkinsHelper.groovy

```diff
@@ -0,0 +1,24 @@
+package org.typo3.chefci.helpers
+
+class JenkinsHelper implements Serializable {
```

Comment:
`JenkinsHelper` to have all Jenkins-related stuff (including things like "give me an input dialog that gracefully times out" and "does this Jenkins job exist"), or should we separate it a bit more into e.g. something that's strictly related to pipelines?